package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Seamless integration between Rollup and PostCSS",
   "main": "dist/index.js",
```

Comment:
`"type": "types/index.d.ts",`

Reply:
It;s already specified on line 33. The file is not published to the registry, so when its installed tools aren't able to find the file, which is why I added it to `files`